documentation/content/en/articles/committers-guide/_index.adoc

```diff
@@ -2750,7 +2751,7 @@ If you have code in the tree with the advertising clause, please consider removi
 In fact, please consider using the above license for your code.
 
 The FreeBSD project discourages completely new licenses and variations on the standard licenses.
-New licenses require the approval of the {core-email} to reside in the main repository.
+New licenses require the approval of {core-email} to reside in the `src` repository.
```

Comment:
Under <https://docs.freebsd.org/en/articles/committers-guide/#pref-license>, the meaning of this line is unclear (there are three repositories): 

> New licenses require the approval of the [core@FreeBSD.org](mailto:core@FreeBSD.org) to reside in the main repository.

Does that mean, the `src` repository?

Reply:
[Yes](https://github.com/freebsd/freebsd-doc/pull/182#discussion_r1186858638).